variables.tf

```
@@ -27,6 +27,11 @@ variable "branch" {
   description = "The git branch to trigger the TFE workspace for"
 }
 
+variable "tfc_agent_role_configuration" {
+  type        = map(string)
+  description = "Configuration for TFC Cloud Agent support"
+}
```

Unless there is any other configuration we need about the TFC Agent setup, this could just be `tfc_agent_role`:

```suggestion
variable "tfc_agent_role_arn" {
  type        = string
  default     = null
  description = "IAM role ARN used by Terraform Cloud Agent to assume role in the created account"
}
```